exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/_metric_exporter/__init__.py

```diff
@@ -169,9 +169,13 @@ def _translate_data(
         )
 
     def export(
-        self, metrics: Sequence[Metric], *args, **kwargs
+        self,
+        metrics: Sequence[Metric],
```

Comment:
I might've missed the discussion on this but why must be pass in by name but can't have `*args` in the signature?

Reply:
It can be passed pass in by position with this signature. Once we go stable, any **new** arguments must be passed by name. If a new argument is added to this function, we would do it like this:

```py
 def export(
        self,
        metrics: Sequence[Metric],
        timeout_millis: float = 10_000,
        *,
        optiona: str = "foo",
        **kwargs,
    ) -> MetricExportResult:
```

the reason is,
- if you call a function with the original signature like `export(metrics, timeout_millis, optiona)`, it will raise a type error because it didn't expect that position arg
- OTOH if you added `*args` to the original signature, you have to decide to put it either before or after `timeout_millis`
    ```py
     def export(
        self,
        metrics: Sequence[Metric],
        # *args before
        timeout_millis: float = 10_000,
        # *args after
        optiona: str = "foo",
        **kwargs,
    ) -> MetricExportResult:
    ```
    
    If you put *args before, then `timeout_millis` has to be passed by name (*args would capture all remaining positions). If you put *args after, then `timeout_millis` has to be passed by position (all args before `*args` have to be positional).

Reply:
Nice thanks for the explanation.